environment.yml

```diff
@@ -20,3 +20,4 @@ dependencies:
   - h5py
   - pp-sketchlib >=1.5.1
   - graph-tool
+  - requests
```

Comment:
Should flask also be added here? Might not yet be needed in the tests, but good to ensure everything can be installed